src/models/link/metadata.rs

```diff
@@ -14,6 +14,7 @@ use crate::Result;
 use crate::models::{Link, Metablock, Metadata, TargetDescription, VirtualTargetPath};
 
 /// Helper to construct `LinkMetadata`.
+#[derive(Clone)]
```

Comment:
I'm trying to follow this change, is this related to #33 or?

Reply:
This is used in test case for me to create a shared example for `MetadataBuilder` without move. I finally found its unnecessary while I could create a `Metadata` from `MetaDatabuilder`, while the former is already cloneable.